### DISCUSSION TITLE
我的 ETH 被盗了 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#36] [Announcements]
# 我的 ETH 被盗了 - forecho's Blog

引言
最近 ETH 跌了，昨天晚上睡觉前看了一下自己的 ETH 钱包，结果看到一个转账通知，钱包内的 0.76 个 ETH 全部被转走了，转走的时间是前天晚上。
这篇文章主要是记录一下自己的经历，以及如何避免自己的钱包被盗。
事件经过

https://blog.forecho.com/my-eth-has-been-stolen.html

**Comment by tumutanzi** [comment]
你这个错误有点低级……

**Comment by yuexuan** [comment]
同感，我的也被盗了，丢失了0.05个。https://etherscan.io/tx/0x9a51ac9cdeaa55f67ef5b000bcb87778faaa1a80423c48de0ab14526cec569d5，特此铭记。
一开始做开发的时候，没有在意，随便创建了一个钱包，就写起来了。很长时间没有问题。
突然有一天，就发现账号被清零了。
没有实体的金钱的感觉，就会很容易忘记，虚拟币不重要。忘记何时会泄露私钥。
所以开发的时候，就用临时的，不超过10刀来测试，自己的强制放在交易所也不得超过100刀，用完了立马落袋。
